cve_bin_tool/format_checkers.py

```diff
@@ -80,6 +80,21 @@ def update_checker_table(file_path, markdown):
         f.writelines(lines)
 
 
+def update_number_of_checkers(file_path, number_of_checkers):
+    """Updates README.md with the new number of checkers"""
+    with Path(file_path) as readme_file:
+        readme_text = readme_file.read_text()
+        readme_text = re.sub(
+            r"(<!-- NUMBER OF CHECKERS START-->)[^<]*(<!--NUMBER OF CHECKERS END-->)",
+            r"<!-- NUMBER OF CHECKERS START-->{}<!--NUMBER OF CHECKERS END-->".format(
+                number_of_checkers
+            ),
```

Comment:
```suggestion
            f"<!-- NUMBER OF CHECKERS START-->{number_of_checkers}<!--NUMBER OF CHECKERS END-->"
```

We use `f-strings` instead of `.format` throughout the codebase